core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -556,6 +556,11 @@
       "Archive any files/directories instead of moving to the HDFS trash or deleting."),
   GC_TRACE_PERCENT("gc.trace.percent", "0.01", PropertyType.FRACTION,
       "Percent of gc cycles to trace"),
+  GC_USE_FULL_COMPACTION("gc.post.metadata.action", "compact", PropertyType.GC_POST_ACTION,
```

Comment:
It'd be nice to make the default for this `flush` in 2.x versions. Keeping it `compact` here is probably a good choice to not introduce behavior changes in a patch release.